src/shell-interface.h

```diff
@@ -142,7 +142,7 @@ struct ShellExternalInterface : ModuleInstance::ExternalInterface {
       std::cout << "exit()\n";
       throw ExitException();
     }
-    std::cout << "callImport " << import->name.str << "\n";
```

Comment:
Done.  

I guess there isn't yet a test for this case, but I manually verified it produces the expected result: e.g. "Fatal: callImport: unknown import: env.abort".